### PR TITLE
Feature/exit code handler

### PR DESCRIPTION
**Pull request by rueyaa332266**
Reopen PR #44 by using another account.

Fix review from rubocop.

### CODE REVIEW

File: lib/bucky/test_equipment/test_case/abst_test_case.rb [hunk 1]
```diff
@@ -8,6 +8,8 @@ module TestEquipment
     module TestCase
```
Comment (naotospace) [inline lib/bucky/test_equipment/test_case/abst_test_case.rb]: It is not necessary to fix this file, maybe.

File: lib/bucky/core/test_core/test_manager.rb [hunk 2]
```diff
@@ -26,13 +27,19 @@ def parallel_new_worker_each(data_set, max_processes, &block)
...
+              Bucky::Core::TestCore::ExitHandler.instance.raise unless $CHILD_STATUS.exitstatus.zero?
+            end
             # Workers decrease when start working
             available_workers -= 1
             fork { block.call(data) }
           end
-          Process.waitall
+          # Handle all exit code in waitall
+          Process.waitall.each do |statuse|
+            Bucky::Core::TestCore::ExitHandler.instance.raise unless statuse[1].exitstatus.zero?
```
[the remark this answers is not in the document]
Comment (rueyaa332266) [reply lib/bucky/core/test_core/test_manager.rb]: Change to child

File: lib/bucky/test_equipment/test_case/abst_test_case.rb (hunk 1 above)
Comment (rueyaa332266) [reply lib/bucky/test_equipment/test_case/abst_test_case.rb]: You are totally right!
Thanks a lot.

**Comment by rueyaa332266** [comment]
I will add the system test.

**Comment by rueyaa332266** [comment]
About System test:
E2E
Because test specification already had the expect of exit code.
So I just add the checking in bat code.
And fix a little description in test specification.

Linkstatus
Add a new case for checking  exit code 1.
Add add the checking exit code 0 in exist bat code.
And fix a little description in test specification.

**Pull request merged**